### DISCUSSION TITLE
Reading ANSII escape sequences

### DISCUSSION THREAD

**Discussion by avidseeker** [jgm/pandoc#10027] [Ideas]
$ ls --color=always
```
dir
dir2
executable
file2.txt
file.txt
```
$ ls --color=always | $EDITOR
```sh
^[[0m^[[01;34mdir^[[0m
^[[01;34mdir2^[[0m
colors.ansii
^[[01;32mexecutable^[[0m
file2.txt
file.txt
```
Can pandoc read these ANSII escape sequences? I'm not sure if they're used in man pages or other document format, but if so, then I guess pandoc could handle them. E.g:

$ echo "\u001b[41m Hello World \u001b[40m" | pandoc -f magic_ftype -t html
```html
<span style="background-color:rgb(187,0,0)"> Hello World </span>
```

Ofc, ls is not the only program for this. There is `git`, `diff`, `grep`, `tree`, ...

Edit: there seem to be ANSII writer PR: https://github.com/jgm/pandoc/pull/9565

**Comment by jgm** [comment]
sorry, no.

**Reply by avidseeker** [reply]
There's already a PR for ANSII writer (https://github.com/jgm/pandoc/pull/9565). So why isn't an ANSII reader within the scope of Pandoc project?
